server-function/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@binaris/shift-server-function",
+  "version": "0.0.0",
```

Comment:
`0.0.1`?

Reply:
Will automatically roll to `0.0.1` when published.